javascript/testing/README.md

```diff
@@ -201,10 +204,13 @@ And here is a functional component that does the same thing using the `useState`
 ```javascript
 export const ClickCounterHooks = () => {
   const [count, setCount] = useState(0)
+
+  const handleButtonClick = () => this.setState(({count}) => ({count: count + 1}))
```

Comment:
I think the "this.setState" part only applies for the class component example and not for the functional example. Isn't it? You probably meant:
 const handleButtonClick = () => setCount(count => count + 1)

Reply:
Yes! copy paste mistake from previous example, thanks!